apps/src/aichat/views/AichatView.tsx

```diff
@@ -90,7 +90,7 @@ const AichatView: React.FunctionComponent = () => {
 
   return (
     <div id="aichat-lab" className={moduleStyles.aichatLab}>
-      {!levelAichatSettings?.hidePresentationPanel && (
+      {!levelAichatSettings?.hidePresentationPanel && hasPublished && (
```

Comment:
Should a `useEffect` be added so that when level changes, `hasPublished` is updated? Currently, if a user goes from a level that has a completed model card and publishes and then goes to a level with an incomplete model card, the presentation mode and view mode toggle are still displayed.

Reply:
^ yep, this. We just need to make sure we're reading from the current project sources and/or setting it in aichatRedux whenever the level changes